metricbeat/module/haproxy/info/info.go

```diff
@@ -44,17 +45,21 @@ func init() {
 // MetricSet for haproxy info.
 type MetricSet struct {
 	mb.BaseMetricSet
+	*helper.HTTP
 }
 
 // New creates a haproxy info MetricSet.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	return &MetricSet{BaseMetricSet: base}, nil
+	http, err := helper.NewHTTP(base)
```

Comment:
We should add `settings: ["ssl", "http"]` to `metricbeat/module/haproxy/_meta/fields.yml` so documentation is updated to mention that http and ssl options are supported.

Reply:
Fixed.